run_linux.go

```diff
@@ -2850,6 +2854,22 @@ func (b *Builder) cleanupRunMounts(context *imagetypes.SystemContext, mountpoint
 			prevErr = err
 		}
 	}
+	// unlock if any locked files from this RUN statement
+	for _, path := range artifacts.LockedTargets {
+		if _, err := os.Stat(path); err == nil {
```

Comment:
Shouldn't ignore an error here, as it could be a symptom of a bigger problem.

Reply:
Added to error log, but we might not want to break loop here since other targets must be unlocked.